tests/bin/check_tron_jobs_test.py

```diff
@@ -1317,6 +1317,44 @@ def test_guess_realert_every(self):
         realert_every = check_tron_jobs.guess_realert_every(job_runs)
         assert_equal(realert_every, 4)
 
+    def test_guess_realert_every_same_next_run_and_prev_run_time(self):
+        job_runs = {
+            'status':
+                'running',
+            'next_run':
+                time.strftime(
+                    '%Y-%m-%d %H:%M:%S',
+                    time.localtime(time.time() + 600),
+                ),
+            'runs': [
+                {
+                    'id':
+                        'MASTER.test.2',
+                    'state':
+                        'failed',
```

Comment:
Instead of making a separate test, can you add 'run_time' to `'MASTER.test.3'` in the test above? That's the usual case for this error, the first run in the list is the scheduled run, which has the same run_time as next_run.